.travis.yml

```diff
@@ -98,8 +96,19 @@ install:
       TAR_OPT="--strip-components=1 -xj";
       mkdir cmake-dist && curl --silent --fail --show-error --location "${CMAKE_DIST_URL}" | tar -C cmake-dist ${TAR_OPT} && export PATH=$PWD/cmake-dist/bin:$PATH;
     fi
-  - if [[ "${JOB_NAME}" == java_test || "${JOB_NAME}" == cmake* ]]; then
-      java -version && echo "JAVA_HOME=${JAVA_HOME}";
+  - |
```

Comment:
before_install is now redendant, right? Can't we remove it?

Reply:
Yup